docs/data/tree-view/getting-started/getting-started.md

```diff
@@ -1,7 +1,7 @@
 ---
 productId: x-tree-view
 title: Tree View - Getting started
-components: SimpleTreeView, RichTreeView, TreeItem, TreeView
+components: SimpleTreeView, RichTreeView
```

Comment:
We removed those APIs in #15093. We missed this.

Reply:
Don't we still have `TreeItem`? 🤔 
https://next.mui.com/x/api/tree-view/tree-item/

Reply:
Oh, got it, I didn't realize.